<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Memory</class>
 <widget class="QDialog" name="Memory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Purisa</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BackMem">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>195</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/back.png</normaloff>:/Images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>280</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/BuffLogo.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>58</y>
     <width>135</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalStorageLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>106</y>
     <width>135</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bytesAvailableLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>130</y>
     <width>135</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>94</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Free Space:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>160</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="volumeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>64</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Volume:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="volumeComboBox">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>20</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="rootPathLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>82</y>
     <width>135</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>106</y>
     <width>113</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Total Storage:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>82</y>
     <width>85</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Root Path:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>58</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
